mage_ai/orchestration/db/models/base.py

```diff
@@ -76,14 +74,16 @@ def save(self, commit=True) -> None:
                 raise e
 
     def update(self, **kwargs) -> None:
+        commit = kwargs.pop('commit', True)
```

Comment:
cc: @wangxiaoyou1993 is this a safe change to make? it seems like it should be similar to the `create` method

Reply:
Yes, i think this is a safe change.